src/components/TextField/TextField.tsx

```diff
@@ -81,16 +81,16 @@ const TextField = React.forwardRef<HTMLInputElement, Props>(
       </React.Fragment>
     );
 
-    const IconWrapper: FC = ({ children }) => (
-      <div css={iconWrapperStyle({ label, rightIcon, leftIcon })}>{children}</div>
+    const IconWrapper: FC<{ iconPosition: 'left' | 'right' }> = ({ children, iconPosition }) => (
```

Comment:
optional: You can expose a type for iconPosition to reuse it in styles.

Reply:
it was only one and didnt want to mess for one property type and redirect the user back and forth to check the reference.